python/tach/sync.py

```diff
@@ -90,9 +85,9 @@ def sync_project(
             project_config=project_config,
             exclude_paths=exclude_paths,
         )
-
+    reorder_modules(project_config)
     tach_yml_content = dump_project_config_to_yaml(project_config)
```

Comment:
Instead of adding this logic as a separate function and calling it in `tach sync`, it would be better to include this behavior in `dump_project_config_to_yaml`, so that we get consistent sorting whenever we write the config file to disk. You'll see there is already a normal sort of the modules happening there, so you could use a custom comparator function to special-case the root module, or simply move the root module to the end after the sort is done.

Reply:
Yes I think that I can work on.